src/modules/pages/components/RouteLayouts/ColonyNavigation.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import UserNavigation from './UserNavigation';
+
+import { RouteComponentProps } from './index';
+
+import styles from './ColonyNavigation.css';
+import HistoryNavigation from './HistoryNavigation';
+
+interface Props {
+  routeProps?: RouteComponentProps;
+}
+
+const displayName = 'pages.RouteLayouts.ColonyNavigation';
+
+const ColonyNavigation = ({ routeProps = {} }: Props) => (
+  <div className={styles.main}>
+    <div>
+      {routeProps.hasBackLink !== false && (
```

Comment:
if hasBackLink will be undefined, we will get true here, do we want this?

Reply:
undefined !== false // true

Reply:
I think this is all handled by the `Default` layout as well, so most likely there's no need for this

Reply:
Fixed